docs/flight-methology.md

```diff
@@ -0,0 +1,70 @@
+---
+id: flight-methology
+title: Flight Metholog
+sidebar_label: Flight Metholog
+---
+
+## Edenia agile metology!
+
+
+We at Edenia are a company that uses agile methodologies that can be adapted to our flexible way of working.
+Since we are not rigid when it comes to change and uncertainty, so we use a hybrid of different agile methodologies applied and tropicalized to our needs in processes.
```

Comment:
Maybe we can include something like this

we also like to be innovative not just with our software solutions but with our processes , constantly evolving and improving the way our team is organized to optimize how we deliver results.